### PR TITLE
Adds tags to cms and card

### PR DESCRIPTION
**Pull request by swemail**
Adds possibility to add tags in the cms and set them on exercises
<image src="https://user-images.githubusercontent.com/1139207/212350059-074598a0-d51e-436c-bd71-7101405b5832.png" width="200" />

### CODE REVIEW

File: client/src/lib/components/Bylines/Byline.tsx [hunk 1]
```diff
@@ -30,23 +30,25 @@ type BylineProps = {
...
-  );
-};
+    return (
+      <Container>
+        <ImageContainer>
+          {pictureURL && <Image source={{uri: pictureURL}} />}
+        </ImageContainer>
+        <Spacer4 />
+        <WrapText numberOfLines={2}>
+          {t('with')} {name} {duration ? `· ${duration} min` : ''}
```
Comment (gewfy) [inline client/src/lib/components/Bylines/Byline.tsx:47]: Not your change. But `min` needs to be translated

File: client/src/lib/content/hooks/useTagsById.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  return useCallback(
+    (id: any) =>
+      t(id, {
+        returnObjects: true,
+      }) as Tag,
+    [t],
+  );
+};
+
+export default useGetTagById;
```
Comment (gewfy) [inline client/src/lib/content/hooks/useTagsById.ts:17]: Nit pick, rename file to `useGetTagById` (singular)

File: client/src/lib/components/Tags/Tags.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <Wrapper key={tag.id}>
+          <Tag key={tag.id} value={tag.tag} />
+          <Spacer4 />
+        </Wrapper>
+      ))}
+    </Container>
+  );
+});
+
+export default Tags;
```
Comment (gewfy) [inline client/src/lib/components/Tags/Tags.tsx:53]: This component is very coupled to the `SessionCard` imo. So I would move it there.

File: client/src/lib/components/Cards/Card.tsx [hunk 4]
```diff
@@ -97,51 +105,51 @@ export const Card: React.FC<CardProps> = ({
...
-    </Wrapper>
-  </>
-);
+}) => {
+  return (
+    <>
+      <Wrapper onPress={onPress}>
+        <ContentWrapper>
+          <LeftCol>
+            <Tags tagIds={tagIds} duration={duration} />
```
Comment (gewfy) [inline client/src/lib/components/Cards/Card.tsx:114]: Me and Kajsa have tried to make the `Card` component very "stupid" and agnostic. Basically that `SessionCard` is the container that resolves all the data that `Card` need to render.
Here we resolve the tags in this component. Imo I would resolve the complete translated tags in `SessionCard` and just pass that as an array that you just render with `Tag` here.
The filtering logic of tags in `Tags` could be moved to a hook specific to `SessionCard` or as a more agnostic `useGetTagsByIds` in `content/`.

File: client/src/lib/components/Tags/Tags.tsx (hunk 3 above)
Comment (gewfy) [reply client/src/lib/components/Tags/Tags.tsx:53]: Or refactor it to a hook as mentioned ⬆️

File: cms/src/collections/collections.ts [hunk 5]
```diff
@@ -48,6 +49,23 @@ export const settings: CmsCollection = {
...
+  slug: '{{id}}',
+  summary: '{{fields.tag}}',
+  extension: 'json',
+  create: true,
+  format: 'json',
+  delete: true,
+  publish: true,
+  i18n: true,
+};
+
```
Comment (gewfy) [inline cms/src/collections/collections.ts:68]: Nice 👌

File: cms/src/fields/tag.ts [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  ID_FIELD,
+  {
+    name: 'tag',
+    label: '🏷 Tag',
+    widget: 'string',
+    i18n: true,
+  },
+  {
+    label: '🔢 Featured order',
+    name: 'featuredOrder',
```
Comment (gewfy) [inline cms/src/fields/tag.ts:14]: Nit pick, can we call it just `order`?

**Review by gewfy — CHANGES_REQUESTED** [review]

File: client/src/lib/components/Tags/Tags.tsx (hunk 3 above)
Comment (swemail) [reply client/src/lib/components/Tags/Tags.tsx]: Did both 😄

**Review by gewfy — APPROVED** [review]
I would have made the tags even more abstract. Basically a an array of strings passed to `Card` where the duration is resolved and added from `SessionCard`. But I leave that up to you. Might change it my self after merge :)

**Comment by swemail** [comment]
> I would have made the tags even more abstract. Basically a an array of strings passed to `Card` where the duration is resolved and added from `SessionCard`. But I leave that up to you. Might change it my self after merge :)

I'll do now then 😂

**Pull request merged**